### ISSUE TITLE
Drill holes into the spoiler board of the CNC Router

### ISSUE THREAD

**Issue by NitramLegov** [comakingspace/do-something#51]

**Comment by LukasFreeze** [comment]
related to #50 - the best way to do this may be to design a CNC pattern for the router to cut into "itself" (only the spoilboard, staying well clear of the frame!) rather than doing it the hard way like Marius Hornberger did in [his video](https://www.youtube.com/watch?v=bpJyqLoqV04).

That will give us high precision and a way to quickly do it again when we exchange the spoilboard.

**Comment by LukasFreeze** [comment]
Maybe we could also think about lowering the spoilboard while we're at it - apparently you can get a Z range of 11 cm by flipping the supports?! -> [photo in Ooznest FB group](https://www.facebook.com/photo.php?fbid=10156523324418630&set=p.10156523324418630&type=3&theater&ifg=1)

**Comment by NitramLegov** [comment]
yes, @Pakue95 mentioned something like this yesterday. I opened #53 for this in order to track the discussion regarding the Z-Range in a separate issue. Nevertheless, #50, #51 and #53 should be tackled together. As of this, they are all linked to the same project.

**Comment by LukasFreeze** [comment]
Good idea that they should be tackled together - I would also propose that
this rather big "luxury" project is done *after* we've worked out the dust
collection (#15), that is much more urgent.

On Jul 19, 2018 5:23 PM, "NitramLegov" <notifications@github.com> wrote:

yes, @Pakue95 <https://github.com/Pakue95> mentioned something like this
yesterday. I opened #53
<https://github.com/comakingspace/do-something/issues/53> for this in order
to track the discussion regarding the Z-Range in a separate issue.
Nevertheless, #50 <https://github.com/comakingspace/do-something/issues/50>,
#51 <https://github.com/comakingspace/do-something/issues/51> and #53
<https://github.com/comakingspace/do-something/issues/53> should be tackled
together. As of this, they are all linked to the same project.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/comakingspace/do-something/issues/51#issuecomment-406315786>,
or mute the thread

**Comment by LukasFreeze** [comment]
Wir sollten noch ein paar Vorbereitungen machen, bevor dafür am Freitag die ganze Zeit draufgeht (und uns nachts einfällt, was wir doch lieber mal gekauft hätten). Ich hoffe ich habe nichts vergessen, was wir in der Telegram-Gruppe angesprochen hatten:

- zum Thema "müssen wir die Alu-Streben unter der Platte beim Löcherfräsen beachten": ich hatte mich ja dafür ausgesprochen, sehe jetzt aber dass unser Vorbild auch [nicht durchgebohrt wurde](https://www.youtube.com/watch?v=bpJyqLoqV04&t=4m28s). Das nehme ich also zurück, solange wir darauf achten nicht zu tief zu gehen ;-)
- halten wir uns beim Rohrdurchmesser und -abstand einfach an Marius Hornberger? Er nimmt 18 mm OD und 75 mm Abstand.
- die alte MDF-Platte sollte laut Wiki 1445 × 1370 × 20 mm haben, ich wäre wieder für das gleiche Format. Eine dünnere Platte wird die Rohre wohl kaum halten können...
- ich sichte morgen gerne mal unseren Multiplex-Vorrat (für die Klemmen/Anschläge), könnte sein dass wir da aber nur zu dünnes Material haben und auch noch etwas kaufen sollten.
- wer holt die Platte(n) und das Rohr? Dafür wäre vermutlich ein großes Auto nötig, oder?
- macht es Sinn und hat jemand Zeit, schon im Vorfeld das Design (teilweise) vorzubereiten?

Falls wir die neue Platte wieder als "Opferplatte" verbraten wollen (von mir aus ungern) wäre damit wahrscheinlich schon genug Arbeit reingesteckt, @emka hatte aber auch noch in der Platte verteilte Gewinde ins Spiel gebracht und ich finde so etwas (oder, evtl einfacher, T-Nuten) als zusätzliche Halteoption nicht schlecht. Gewinde könnte man leicht mit Einschlagmuttern von der Unterseite realisieren, aber eben am besten bevor die Platte montiert wird und deshalb ggf parallel zu diesem Projekt. T-Nuten sollten wir auch später noch nachrüsten können, oder?

Mein Vorschlag für Leute, die gerne die "Freiheit" möchten, direkt in die Unterlage zu schrauben und zu fräsen, wäre in Zukunft das Aufspannen einer separaten Opferplatte. Und wer versehentlich durchfräst könnte mit einem Schild klar darauf hingewiesen werden, dass Ersatz Zeit und Geld kostet und was eine angemessene Spende dafür wäre. Ganz davon abgesehen wie gefährlich es sein kann, so tief durchzufräsen dass die "tabs" erst in der Unterlage anfangen und das Werkstück sich frei bewegen kann (ohne jetzt Namen zu nennen)

**Comment by NitramLegov** [comment]
My two cents:
Das Format der aktuellen MDF-Platte habe ich letzte Woche ausgemessen. die 1445x1370 passem :)

Frage: Brauchen wir wirklich eine neue Platte? Eigentlich ist die aktuelle noch dick genug. Gerade, wenn wir nicht durchfräsen, sollte das eigentlich passen.

Bezüglich Rohr: Im CNC-Raum steht ein Alurohr, das wir nehmen können. Habe ich mal mitgebracht, weil es bei mir daheim nur rumstand (das ist so ein 0815 Ding, das man im Bauhaus kriegt)

Ich bin gegen Gewinde oder generell metall in der Platte. Ob es nun eine Opferplatte ist oder nicht: Das Risiko, dass doch mal einer nicht aufpasst und reinfräst ist immer gegeben...und bei Metall in der Platte wäre mir das Risiko von Funkenbildung und damit Feuer zu groß (Ja, Einschlagmuttern sind an der unterseite, trotzdem find ich das prinzipiell nicht begrüßenswert).

**Comment by LukasFreeze** [comment]
Hmm, wir könnten das ganze Prozedere und vor allem die neuen Klemmen wahrscheinlich schon erstmal mit der alten Platte testen. Allzu lange hält die wahrscheinlich nicht mehr, aber das könnte echt eine gute Möglichkeit sein, diese "Großbaustelle" etwas zu teilen und damit schaffbar zu machen!

**Comment by LukasFreeze** [comment]
Ich wollte ja noch was gegen Alu sagen weil ich denke dass es sich schnell verbiegt, aber dein Rohr ist zum Glück doch sehr stabil. Und komischerweise magnetisch... ;-D

Der Nenndurchmesser dürfte jedenfalls **20 mm** sein, das ist wichtig fürs Design der Klemmen.

Multiplex haben wir leider nur in 6 und 12 mm Dicke, 24 wären da schon noch nett falls es noch jemand zum Baumarkt schafft!

**Comment by LukasFreeze** [comment]
Wir haben zwei 11 mm tiefe, blind endende Testlöcher in die Platte gefräst um die erste Klemme (#50) zu testen. Viel weiter kamen wir nicht, dann kam #104 dazwischen...

Geplant waren jedenfalls 90 mm Lochabstand, wenn wir die Alustreben im jetzigen Setup vermeiden und *durch* die Platte bohren wollten bräuchten wir 120 mm, was die Klemmen nicht hergeben.

**Comment by NitramLegov** [comment]
Closing this stale issue
